onnx_tf/handlers/backend/image_scaler.py

```diff
@@ -0,0 +1,21 @@
+import tensorflow as tf
+
+from onnx_tf.handlers.backend_handler import BackendHandler
+from onnx_tf.handlers.handler import onnx_op
+
+
+@onnx_op("ImageScaler")
+class ImageScaler(BackendHandler):
+
+  @classmethod
+  def version_1(cls, node, **kwargs):
+    input_dict = kwargs["tensor_dict"]
+    x = input_dict[node.inputs[0]]
+    scale = node.attrs["scale"]
```

Comment:
`scale` is not a required attr and has default value 1.0f. 
Should use `node.attrs.get("scale", 1.0)`.